lib/game.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+# Class Game
+class Game
+  attr_accessor :grid
+
+  def initialize(grid)
+    @grid = grid
+  end
+
+  def win_game
+    diagonal || vertical || horizontal
+  end
+
+  # private methods
+  private
+
+  # Diagonals
+  def diagonal
+    left_diagonal || right_diagonal
+  end
+
+  def left_diagonal
+    return true if @grid[0][0] == @grid[1][1] && @grid[1][1] == @grid[2][2]
```

Comment:
:bulb: This will return true/false already `@grid[0][0] == @grid[1][1] && @grid[1][1] == @grid[2][2]`we don't need the `return ... if` here. 
### Do this for all other occurrences.